python/setup.py

```diff
@@ -29,6 +25,18 @@
 import os
 import platform
 
+#
+# Force `setup_requires` stuff like Cython to be installed before proceeding
+#
+from setuptools.dist import Distribution
+Distribution(dict(setup_requires=['cython>=0.21', 'numpy>=1.7.1']))
```

Comment:
The correct name of the package is `Cython` – using the lowercase version will result in issues with some tooling as it is not the canonical name.